package.json

```diff
@@ -29,6 +30,12 @@
     ]
   },
   "wireit": {
+    "bidi-types": {
+      "command": "tools/generate-bidi-types.sh",
```

Comment:
If we specify the input file in `files`, then this would work with `--watch`. Maybe it's overkill and only useful when one is iterating on the CDDL, commenting for completeness anyway.

Reply:
@OrKoN wants this command to just be a utility command for easily regenerating on our end. The anticipated workflow is:

 - You have `w3c/webdriver-bidi` cloned somewhere.
 - You generate the CDDL document there.
 - You run `npm run bidi-types -- path/to/all.cddl`

This decouples generation with building and allows us to use a custom CDDL when needed.